timur/lib/client/jsx/components/model_viewer.jsx

```diff
@@ -1,49 +1,63 @@
 import React from 'react';
-import { connect } from 'react-redux';
+import {connect} from 'react-redux';
 import TableViewer from './table_viewer';
 
-import { selectTemplate, selectDocuments, displayAttributes } from '../selectors/magma';
+import {
+  selectTemplate,
+  selectDocuments,
+  displayAttributes
+} from '../selectors/magma';
 import AttributeViewer from './attributes/attribute_viewer';
 
-const ModelViewer = ({ record_names, documents, template, attribute_names, page, pages, page_size, setPage }) => {
-  if (!template || !record_names) return <div className='table'/>;
+const ModelViewer = ({
+  record_names,
+  documents,
+  template,
+  attribute_names,
+  page,
+  pages,
+  page_size,
+  setPage,
+  restricted_attribute_names = null
+}) => {
+  if (!template || !record_names) return <div className='table' />;
 
   record_names = Object.keys(documents).sort();
 
-  let columns = attribute_names.map(
-    att_name => (
-      {
-        Header: att_name,
-        accessor: att_name,
-        Cell: ({value, original}) => <AttributeViewer
-          template={template}
-          document={original}
-          value={value}
-          attribute={ template.attributes[att_name] }
-        />
-      }
+  const display_attribute_names = restricted_attribute_names
```

Comment:
Add a parameter to `model_viewer` to only display certain attributes from the model template.